cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -5,32 +5,55 @@ import { AwsDefaults } from '../default-service/AwsDefaults.service';
 import { STORE_SECTION_FORM } from '../../../../../state-management/reducers/Form.reducer';
 import { AWS_ADD_RESOURCES } from '../../../../../state-management/actions/Resources.actions';
 import { AwsResourceAction, FormAction, StoreDispatch } from '../../../../../shared/types/types';
+import { AWSKeyPair } from '../../../../../swagger-api/models/AWSKeyPair';
 interface AwsOrchestratorProps {
     awsState: { [key: string]: any };
     awsDispatch: StoreDispatch;
+    errorMessage: { [key: string]: any };
```

Comment:
nit: when I saw `errorMessage` I was expecting the type to be a string, rather than an object associating keys to objects. I'm thinking this is the error object we generally use, so that the key is actually a field name?
If so, what do you think of declaring this as
`errorObject: { [fieldName: string]: any };`?

Reply:
Yes sir. You are right. Correct it right away